code/game/machinery/slotmachine.dm

```diff
@@ -71,6 +76,9 @@
 		playsound(src.loc, 'sound/machines/ding.ogg', 50, 1)
 		addtimer(CALLBACK(src, PROC_REF(spin_slots), usr.name), 25)
 
+		if(emagged)
+			emagged_spinning(usr)
```

Comment:
```suggestion
			emagged_spinning(ui.user)
```
this should use `ui.user` which can be accessed by adding the `datum/tgui/ui` parameter to this proc